spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/services/impl/DefaultTaskServiceTests.java

```diff
@@ -192,9 +192,9 @@ public void createSequenceComposedTask() {
 	@Test
 	@DirtiesContext
 	public void createSplitComposedTask() {
-		taskService.saveTaskDefinition("splitTask", "<AAA || BBB>");
-		verifyTaskExistsInRepo("splitTask",
-				"<splitTask-AAA || splitTask-BBB>");
+		String dsl = "<AAA || BBB>";
+		taskService.saveTaskDefinition("splitTask", dsl);
```

Comment:
Would be good if we also have some tests for `taskService.isComposedDefinition(dsl)` as well. Will add when merging.